build.sbt

```diff
@@ -146,19 +147,24 @@ lazy val runtimeJSDOMTest =
 
 lazy val plugin =
   project
-    .dependsOn(runtimeJVM % Test)
+    // we need both runtimes compiled prior to running tests
+    .dependsOn(runtimeJVM % Test, runtimeJS % Test)
```

Comment:
Aha, it was some trickiness actually. No regression.

Reply:
Ah this make sense. 👍🏼 good catch.